source/fundamentals/data-formats/custom-types.txt

```diff
@@ -282,157 +276,145 @@ The ``fallback_encoder`` Callable
+   class MyStringType(object):
+       def __init__(self, value):
+           self.__value = value
 
+       def __repr__(self):
+           return "MyStringType('%s')" % (self.__value,)
 
-  class MyNumberType(object):
-      def __init__(self, value):
-          self.__value = value
+   class MyNumberType(object):
+       def __init__(self, value):
+           self.__value = value
 
-      def __repr__(self):
-          return "MyNumberType(%s)" % (self.__value,)
+       def __repr__(self):
+           return "MyNumberType(%s)" % (self.__value,)
 
-We also define a fallback encoder that pickles whatever objects it receives
+Define a fallback encoder that pickles whatever objects it receives
 and returns them as ``~bson.binary.Binary`` instances with a custom
-subtype. The custom subtype, in turn, allows us to write a TypeDecoder that
-identifies pickled artifacts upon retrieval and transparently decodes them
+subtype. The custom subtype allows you to write a TypeDecoder that
+identifies pickled artifacts on retrieval and transparently decodes them
 back into Python objects:
 
 .. code-block:: python
 
-  import pickle
-  from bson.binary import Binary, USER_DEFINED_SUBTYPE
+   import pickle
+   from bson.binary import Binary, USER_DEFINED_SUBTYPE
 
 
-  def fallback_pickle_encoder(value):
-      return Binary(pickle.dumps(value), USER_DEFINED_SUBTYPE)
+   def fallback_pickle_encoder(value):
+       return Binary(pickle.dumps(value), USER_DEFINED_SUBTYPE)
 
 
-  class PickledBinaryDecoder(TypeDecoder):
-      bson_type = Binary
-
-      def transform_bson(self, value):
-          if value.subtype == USER_DEFINED_SUBTYPE:
-              return pickle.loads(value)
-          return value
+   class PickledBinaryDecoder(TypeDecoder):
+       bson_type = Binary
 
+       def transform_bson(self, value):
+           if value.subtype == USER_DEFINED_SUBTYPE:
+               return pickle.loads(value)
+           return value
 
 .. note::
 
-  The above example is written assuming the use of Python 3. If you are using
-  Python 2, ``bson_type`` must be set to ``Binary``. See the
-  :ref:`decoding-binary-types` section for a detailed explanation.
-
+   The preceding example assumes the use of Python 3. If you are using
+   Python 2, you must set ``bson_type`` to ``Binary``. See the
+   :ref:`decoding-binary-types` section for a detailed explanation.
 
-Finally, we create a ``CodecOptions`` instance:
+Finally, create a ``CodecOptions`` instance:
 
 .. code-block:: python
 
-  codec_options = CodecOptions(
-      type_registry=TypeRegistry(
-          [PickledBinaryDecoder()], fallback_encoder=fallback_pickle_encoder
-      )
-  )
+   codec_options = CodecOptions(
+       type_registry=TypeRegistry(
+           [PickledBinaryDecoder()], fallback_encoder=fallback_pickle_encoder
+       )
+   )
 
-We can now round trip our custom objects to MongoDB:
+You can now round trip your custom objects to MongoDB:
```

Comment:
another page i did used "round-trip" with a hyphen. i'm not sure which is correct, but when it's hyphenated, it's clearer to me that it's intended as a single verb

```suggestion
You can now round-trip your custom objects to MongoDB:
```